yarn-project/aztec-sandbox/README.md

```diff
@@ -1 +1,52 @@
 # aztec-sandbox
+
+Aztec Sandbox is a package that allows for a simple development environment on Aztec stack. It creates an Aztec RPC server that listens for HTTP requests on `localhost:8080` by default. When started, it deploys all necessary L1 Aztec contracts and then starts listening for RPC requests.
+
+## How to run:
+
+### Docker Compose
+
+The easiesty way to run is by using `docker compose up`. This will create two containers:
+
+1. The sandbox listening on port `8080`
+2. An anvil instance listening on port `8545`
+
+### Node Server
+
+You can also run it as a standalone node server with:
+
+```sh
+yarn start
```

Comment:
worth mentioning that this wont run without running ./bootstrap.sh on circuits to create the wasm bundle